tests/io/v3/was/attachments/test_attachments_api.py

```diff
@@ -0,0 +1,120 @@
+'''
+Testing the Attachments endpoints
+'''
+import os
+import re
+
+import responses
+
+ATTACHMENTS_BASE_URL = r'https://cloud.tenable.com/api/v3/was/attachments'
+USERS_API_ID = r'([0-9a-fA-F\-]+)'
+
+dir_name = os.path.dirname(os.path.abspath(__file__))
+
+IMG_ATTACHMENT_FILE = 'img_attachment_file.png'
+TXT_ATTACHMENT_FILE = 'text_attachment_file.txt'
+
+IMG_ATTACHMENT_FILE_PATH = os.path.join(dir_name, IMG_ATTACHMENT_FILE)
+TXT_ATTACHMENT_FILE_PATH = os.path.join(dir_name, TXT_ATTACHMENT_FILE)
+
+IMG_ATTACHMENT_ID = 'b1b335cf-938f-462a-b83d-088b72428a2a'
+TXT_ATTACHMENT_ID = '49484c93-baa5-4952-8043-9afad238cb7b'
+
+IMG_CONTENTS = None
+TXT_CONTENTS = None
+
+
+def generate_image_contents():
+
+    global IMG_CONTENTS
+
+    if IMG_CONTENTS is None:
+        with open(IMG_ATTACHMENT_FILE_PATH, 'rb') as img_file:
+            IMG_CONTENTS = img_file.read()
+
+
+def generate_text_contents():
+
+    global TXT_CONTENTS
+
+    if TXT_CONTENTS is None:
+        with open(TXT_ATTACHMENT_FILE_PATH, 'rb') as txt_file:
+            TXT_CONTENTS = txt_file.read()
+
+
+@responses.activate
+def test_download_img_attachment(api):
+
+    generate_image_contents()
```

Comment:
You could call generate_image_contents and generate_text_contents more cleanly as a fixture. Ref: https://github.com/dhvanilshah-tenable/pyTenable/pull/39/files#diff-fa3961de3a9de1ad4460055d8ba0cedd004c7c1355d983724362fdeba568cfa7R8

Reply:
Done.